test-framework/test-suites/integration/tests/report/test_report_host.py

```diff
@@ -0,0 +1,71 @@
+import pytest
+
+
+@pytest.mark.usefixtures("revert_database")
```

Comment:
Don't need this

Reply:
Sorry, I should be more specific. You don't need the usefixtures line because "revert_database" is always included via the pytest configuration.